src/content/docs/custom-pipeline/setup.mdx

```diff
@@ -1,209 +1,186 @@
+shortcut, `` ` `` is on the left of the enter key). Let's verify that tools are available and
+working by running some quick and simple tests commands.
 
-The project uses _poetry_ to manage python dependencies. To install it using pipx,
-run the following commands :
+#### nf-core/tools
 
 ```bash
-pip install pipx
-pipx ensurepath
-pipx install poetry==1.8.*
+nf-core modules list remote
 ```
 
-:::note
-If the second command above fails, `pipx` cannot be found in the path. Prepend the
-second command with `$(which python) -m` and rerun the whole block.
-:::
+Check the command's header, above the list of modules, you should see this info message :
 
-:::caution
-Poetry doesn't like when other python environments are activated around it. Make
-sure to deactivate any before calling `poetry` commands.
-:::
+```bash frame="none"      
+INFO    Modules available from https://github.com/scilus/nf-neuro.git (main)
+```
 
-Once done, install the project with :
+#### Docker
 
 ```bash
-poetry install --no-root
+docker run hello-world
 ```
 
-### Loading the project's environment
+Check the command's output, it should be the same as when you run the command from outside the container.
 
-:::caution
-Make sure no python environment is activated before running commands !
-:::
-
-The project scripts and dependencies can be accessed using :
+#### Nextflow
 
 ```bash
-poetry shell
+nextflow run helloworld
 ```
 
-which will activate the project's python environment in the current shell.
+Verify the **nextflow-io/helloworld** pipeline is pulled successuflly. Check the command's output,
+you've been greeted in 4 different languages (english, french, italian, spanish) !
 
-:::note
-You will know the poetry environment is activated by looking at your shell. The
-input line should be prefixed by : `(nf-neuro-tools-py<version>)`, with `<version>`
-being the actual Python version used in the environment.
-:::
+#### nf-test
 
-To exit the environment, simply enter the `exit` command in the shell.
+```bash
+nf-test test modules/nf-neuro/reconst/dtimetrics -profile docker
+```
 
-:::caution
-Do not use traditional deactivation (calling `deactivate`), since it does not relinquish
-the environment gracefully, making it so you won't be able to reactivate it without
-exiting the shell.
-:::
+This test is longer, but quite effective at verifying all tools communicate well together. You'll
+have to wait around 5 minutes for it pull docker containers and run. <span class="hover:text-yellow-700">
+Enough time to finish your hot drink <CoffeeIcon class="inline-icon" aria-hidden /></span> !
 
-### Global environment
+On completion, check the command output, it should end with <span class="text-emerald-400">SUCCESS</span>
+and all test cases labeled as <span class="text-emerald-400">PASSED</span>.
 
-Set the following environment variables in your `.bashrc` (or whatever is the equivalent for your shell) :
+### Nextflow language server
```

Comment:
This is kind of an extra after the validation of the setup. I'm not sure we want to go further.

Reply:
Maybe find a way to differentiate the next parts from the real setup.

Reply:
Maybe we could put the sections on the language server and data access in the **How-to** / **Advanced Tutorials**. There is place for subsections and would not break the flow.